WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -231,7 +231,11 @@ def create
       registration_events_attributes: [:id, :event_id, :_destroy],
     ]
     if current_user.can_manage_competition?(competition_from_params)
-      permitted_params << :status
+      permitted_params << :accepted_at
+      status = params[:registration][:status]
+      if status
```

Comment:
I'm confused, why did all the tests need to change if you wrote this code to present the same HTTP api?